### PR TITLE
enhance: skip loading duplicated index

### PR DESCRIPTION
**Pull request by yah01**
this protect the loading index from failure, and speed up the loading progress

### CODE REVIEW

**Comment by yah01** [comment]
rerun ut

**Comment by yah01** [comment]
rerun ut

**Comment by samhuang-z** [comment]
rerun ut

File: internal/querynodev2/segments/segment.go [hunk 1]
```diff
@@ -883,19 +883,27 @@ func (s *LocalSegment) LoadDeltaData(ctx context.Context, deltaData *storage.Del
...
+	if old != nil && old.IndexInfo.GetIndexID() == indexInfo.GetIndexID() {
+		log.Warn("index already loaded")
+		return nil
+	}
+
+	loadIndexInfo, err := newLoadIndexInfo(ctx)
+	defer deleteLoadIndexInfo(loadIndexInfo)
+	if err != nil {
+		return err
+	}
```
Comment (congqixia) [inline internal/querynodev2/segments/segment.go:905]:
```suggestion
	loadIndexInfo, err := newLoadIndexInfo(ctx)
	if err != nil {
		return err
	}
	defer deleteLoadIndexInfo(loadIndexInfo)
```

**Review by congqixia — CHANGES_REQUESTED** [review]

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**